data/categories.txt

```diff
@@ -15033,3 +15074,84 @@ uk:Електростанція
 vi:Nhà máy điện
 zh-Hans:发电厂
 zh-Hant:發電廠
+
+shop-auction|@secondhand
+en:4Auction|Auctioneer|Auction House
+ar:مزاد علني
+be:Аўкцыён
+bg:Търг
+ca:Subhasta
+cs:Aukce
+da:Auktion
+de:Auktion
+el:Δημοπρασία
+es:Subasta
+et:Oksjon
+eu:Enkantea
+fa:حراج
+fi:Huutokauppa
+fr:Vente aux enchères
+he:מכירה פומבית
+hu:Aukció
+id:Lelang
+it:Asta
+ja:オークション
+ko:경매
+mr:लिलाव
+nb:Auksjon
+nl:Veiling
+pl:Aukcja
+pt:Leilão
+ro:Licitație
+ru:Аукцион
+sk:Aukcia
+sv:Auktion
+sw:Mnada
+th:ประมูล
+tr:Müzayede
+uk:Аукціон
+vi:Bán đấu giá
+zh-Hans:拍卖会
+zh-Hant:拍賣
+
+shop-collector|@shopping|@secondhand
+en:5Collectables
+ar:المقتنيات
+be:Калекцыйныя прадметы
+bg:Колекционерски стоки
+ca:Col·leccionables
```

Comment:
Is it a real Catalan translation with a dot?

Reply:
https://www.wikidata.org/wiki/Q208165 (more languages) & https://ca.wikipedia.org/wiki/Col%C2%B7leccionisme
it does seem to be a legitimate translation :)